### ISSUE TITLE
[fltk] dll are not copied to the output directory in debug mode. 

### ISSUE THREAD

**Issue by vindar** [Microsoft/vcpkg#2512]
**Setup**: Windows 10, vs 2017. 
**Package**: FLTK:x86-windows

**Problem**: When I compiling a program using fltk in debug mode, the dlls are **not** copied alongside the executable (whereas they are copied when compiling in release mode). 

Step to reproduce to problem: 

CMakeLists.txt

```
cmake_minimum_required(VERSION 3.0)
project(tfltk)
find_package(FLTK REQUIRED)
add_executable(tfltk main.cpp)
target_link_libraries(tfltk PUBLIC ${FLTK_LIBRARIES})
target_include_directories(tfltk PUBLIC ${FLTK_INCLUDE_DIR})
```

main.cpp

```
#include <FL/Fl.H>
#include <FL/Fl_Window.H>

int main(int argc, char *argv[]) {
	Fl_Window * G_win = new Fl_Window(640, 480, "test");
	G_win->end();
	G_win->show();
	return(Fl::run());
}

```

**Comment by vindar** [comment]
**Additional info:** 

dumpbin on the resulting exe above shows that, when compiling in debug mode, the exe tries to link with the **release** version of fltk's dll:

```
    libfltk_images_SHARED.dll
    libfltk_forms_SHARED.dll
    libfltk_SHARED.dll
```

(i.e. the ones without the d suffix). 

Since these dll are located in `vcpkg\installed\x86-windows\bin` and not in `vcpkg\installed\x86-windows\debug\bin`, this must be the reason why they are not copied over during the build process. So I guess there is a problem with the building script for fltk in debug mode which does not indicate the correct debug libraries...

**Comment by JackBoosY** [comment]
Hi @vindar , this problem has been fixed in #1320. Plz update vcpkg and rebuild.
Thx for post this issue.
